Development-Report.md

```diff
@@ -86,25 +86,37 @@ The software division of the project follows the **MVC (Model-View-Controller)**
 
 -**View**: Represents the visualization of the data that model contains. 
 
--**Controller**: Exists between the view and the model. It listens to events triggered by the view and executes the appropriate reaction to these events.
+-**Controller**: Exists between the Model and View working as intermediate of information.
 
-We will focus on three main components of the project where the first two are inserted in the model category and the third in the view category: 
-- Data information storage: all the information related to then events and it's location 
--  Route computation and map integration 
--  App design and managment of functionalities in pages
+Our project is centered in the implementation of three main parts:
 
-The route computation and the map display is developed with the assistance of the **GoogleMaps API**. This way by giving the correct coordinates of the points of source and destination we can obtain the correct route. After that we need to compute the result and represent it in the map.
+	- Storage of the data;
+	
+	- Route computation with map integration;
+	
+	- App design and functionalities;
 
-The managment of the pages and correspondent functionalities is the section where all the visual components of the pages are implemented. In addiction this section also implies the use of the data storaged as arguments in order to display in the correct places (buttons, bars ,...) and allow to compute the right routes in the section above.
 
+The information from the server needs to be processed and organized into classes present in the package - Model-POI. 
+Those classes allows us to represent the events in the conference and the points of interest associated with them. Furthermore we implemented a graph with connections that allows to represent the paths from point to point. This way we are able to represent the paths available in the building.
+
+Like previously said the graph representation enables us to represent all the paths possible in the building. Hence we can use this representation to compute the routes desired by implementing a simple Dijkstra method. In the end we have a list of connected points of interest from the origin to the destination point to represent in the map. 
+
+The app design is the related to the GUI interface of the application. This includes the associated visual components of the pages and the correct handle of the information requested to present there and also the the respective implementation of all the functionalities associated with the GUI.
```

Comment:
... app design is related to the ...
... correct handling ...
... also the respective ...